### PR TITLE
Us 07 merchant items show page

### LINKED ISSUE #7: US-06-merchant-items-index-page
6. Merchant Items Index Page

As a merchant,
When I visit my merchant items index page (merchants/:merchant_id/items)
I see a list of the names of all of my items
And I do not see items for any other merchant

### PR DESCRIPTION
**Pull request by yainporter**
## Type of change
- [X] New feature
- [ ] Bug Fix
- [X] Test
- [ ] Refactor/Style

## Description
Relevant User Story(s): 
US - 7 and US - 6
*Brief summary of the change(s) included in this pull request*

- Create merchants/items/show.html.erb
- Create additional route
- Turn item names on /merchants/items/index.html.erb to links

## Implements/Fixes Issue(s):
closes #7 

## Neccesary checkmarks:
- [X] All Tests are Passing
- [X] The code will run locally

## Any Outstanding Issues/Problems?
- [ ] My code is wonderful, no issues here!
- [X] Yeah, see additional words below
*Description of problem(s)*
`<%= number_to_currency(@item.unit_price/100) %>` is a calculation, but if I remember correctly, calculations for formatting purposes in the View should be fine
## Any Fun Facts or Gifs You Want to Add to This PR?
The shortest war in history lasted only 38 minutes! It occurred between the countries of Britain and Zanzibar on August 27, 1896. The conflict arose when Sultan Khalid bin Barghash of Zanzibar refused to step down after Britain declared him unfit to rule. British warships quickly bombarded the Sultan's palace, and within less than an hour, the war was over, making it one of the shortest wars ever recorded in history.

### CODE REVIEW

File: app/controllers/merchant/items_controller.rb
Comment (kohljd) [inline app/controllers/merchant/items_controller.rb:8]: If you ever need just a merchant's items on a view later, `@items = @merchant.items` could be useful to refactor in the future

**Review by kohljd — CHANGES_REQUESTED** [review]

File: app/views/merchant/items/show.html.erb
Comment (yainporter) [reply app/views/merchant/items/show.html.erb:3]: I'd like to wait on a confirmation from Juliet or Abdul on this one if it truly breaks MVC, because I think this is a gray area. I'm manipulating data to display on the view page... Modification of data in the Model breaks MVC

File: app/controllers/merchant/items_controller.rb
Comment (yainporter) [reply app/controllers/merchant/items_controller.rb:8]: I don't think I understand... I only want one item here

Comment (yainporter) [reply app/controllers/merchant/items_controller.rb:8]: Oh, I changed to the full page view, I see what you mean now, I didn't make def index, I just accidentally deleted it and had to put it back

**Review by kohljd — APPROVED** [review]
Approved after group discussion

**Pull request merged**
